docs: how to use UI features (#5067)
recognai/rubrix: argilla/docs/how_to_guides/annotate.md

=== "The left pane"

    This area is displayed on top of **the control panel** for performing searches, applying filters and sorting results. The record card(s) are displayed one by one or in a vertical list depending on the active view: **Focus view** or **Bulk view.** A card includes one or many fields and an ellipsis menu to mainly access the record extra info like the metadata.

=== "The right pane"

    This is where you annotate your dataset. Simply fill it out as a form, then choose to Submit, Save as Draft, or Discard your response to send the records to their corresponding queues.

=== "The left bottom panel"

    This expandable area displays the annotation guidelines.

=== "The right bottom panel"

    This expandable area displays your annotation progress.

![UI overview](../assets/images/how_to_guides/annotate/ui_overview.png)

!!! tip
    The app is responsive which enable you to adapt your workspace from two to one column. You can even use your mobile to provides simple feedback on your datasets.

### Shortcuts

Argilla UI includes a range of shortcuts. For the main actions submit, discard, save as draft and the labels the keys are showed in the button.

> To move from a question to another, or between records using the keyboard you may have a look to the table just below.

Reviewer: ```suggestion
To move from one question to another or between records using the keyboard take a look at the table below.
```

!!! tip
    Shortcuts provide a smoother experience, especially with a long list of labels or single-question forms.

> ??? "Consult the available shortcuts"

Reviewer: ```suggestion
??? "Available shortcuts"
```

    | Action | Keys |
    | --- | --- |
    | Activate form | ⇥ Tab |
    | Move between questions | ↓ Down arrow or ↑ Up arrow |
    | Select and unselect label | 1, 2, 3 |
    | Move between labels or ranking options | ⇥ Tab or ⇧ Shift ⇥ Tab |
    | Select rating and rank | 1, 2, 3 |
    | Fit span to character selection | Hold ⇧ Shift |
    | Activate text area | ⇧ Shift ↵ Enter |
    | Exit text area | Esc |
    | Discard | ⌫ Backspace |
    | Save draft (Mac os) | ⌘ Cmd S |
    | Save draft (Other) | Ctrl S |
    | Submit | ↵ Enter |
    | Move between pages | → Right arrow or ← Left arrow |

### View by status

The view selector is set by default on Pending.

> If you are starting an annotation effort, all the records are initially kept in the Pending view. Once you annotate little by little, the records will move to the other queues: Draft, Submitted, Discarded.

Reviewer: ```suggestion
If you are starting an annotation effort, all the records are initially kept in the Pending view. Once you start annotating, the records will move to the other queues: Draft, Submitted, Discarded.
```

> - **Pending**: The records with no responses.

Reviewer: ```suggestion
- **Pending**: The records without a response.
```

> - **Draft**: The records with partial responses. They can be submitted or discarded later. They can’t go back to the pending queue.

Reviewer: ```suggestion
- **Draft**: The records with partial responses. They can be submitted or discarded later. You can’t move them back to the pending queue.
```

> - **Discarded**: The records may or may not have responses. They can be edited but can’t go back to the pending queue.

Reviewer: ```suggestion
- **Discarded**: The records may or may not have responses. They can be edited but you can’t move them back to the pending queue.
```

- **Submitted**: The records have been fully annotated and have already been submitted.

### Suggestions

> If your dataset includes model prediction, you will see them represented by a sparkle icon `✨` in the label or value button. We call them “Suggestions”, and they appear in the form as pre-filled responses. If the suggestions convince you, you just need to click on the Submit button, and they will be considered as another annotation.

Reviewer: ```suggestion
If your dataset includes model predictions, you will see them represented by a sparkle icon `✨` in the label or value button. We call them “Suggestions” and they appear in the form as pre-filled responses. If you agree with the suggestion, you just need to click on the Submit button, and they will be considered as your response.
```

> If the suggestion is incorrect, you can modify it and submit it.

Reviewer: ```suggestion
If the suggestion is incorrect, you can modify it and submit your response.
```

The score per suggested label/value is displayed for MultiLabelQuestion and RankingQuestion.

> You can also order your labels in the preference settings in order to show suggestions first, on top of the label list.

Reviewer: ```suggestion
For label questions, admins can also choose the ordering of labels in the datasets settings. Additionally, admins can choose to always show suggested labels at the beginning of the list.
```

### Focus view

![Focus view](../assets/images/how_to_guides/annotate/focus_view.png)

This is the default view to annotate your dataset linearly, displaying one record after another.

> **You should use this when:** getting acquainted with a dataset or when the annotation team is very diverse, the topic is generic.

Reviewer: @sdiazlor can we turn this into tips to make it more standardized? After that, we'll add some edits to the content itself, ping us on slack once these are turned into tips

> Once you submit your first annotation, the next record will appear automatically. To see again your submitted record, just click on prev.

Reviewer: ```suggestion
Once you submit your first response, the next record will appear automatically. To see again your submitted record, just click on prev.
```
